src/AWS.Deploy.CLI/Commands/CommandFactory.cs

```diff
@@ -308,7 +312,7 @@ private Command BuildListCommand()
                 listCommand.Add(_optionRegion);
                 listCommand.Add(_optionProjectPath);
                 listCommand.Add(_optionDiagnosticLogging);
-            } 
+            }
```

Comment:
nit: can you remove whitespace change?

Reply:
I would rather keep them, there shouldn't be a whitespace once the line has ended.